engine/src/flutter/lib/ui/platform_dispatcher.dart

```diff
@@ -1506,6 +1506,88 @@ class PlatformDispatcher {
   // configurationId does not match any configuration.
   @Native<Double Function(Double, Int)>(symbol: 'PlatformConfigurationNativeApi::GetScaledFontSize')
   external static double _getScaledFontSize(double unscaledFontSize, int configurationId);
+
+  /// Color palette preferred by the operating system UI widgets.
+  ///
+  /// On platforms that do not support a UI color palette this value is null.
+  ///
+  /// As of the current release, system colors are supported on web only.
+  ///
+  /// This is typically used in conjunction with
+  /// [AccessibilityFeatures.highContrast]. In particular, on Windows, when
```

Comment:
this is for weindows web right?

Reply:
This is supported by all browsers on all operating systems. For example, I can get a list of system colors in Safari on macOS (in fact, you will notice that the test I added runs in all browsers on all operating systems).

Reply:
but is the windows web the only platform that user can set it without doing manually modifying the css?

Reply:
Correct. System colors is a general feature of being able to match your UI colors with system colors (https://blog.jim-nielsen.com/2021/css-system-colors/). However, on Windows the user can customize those colors.